CHANGELOG.md

```diff
@@ -11,6 +11,8 @@ Versioning](http://semver.org/spec/v2.0.0.html) except to the first release.
 ### Added
 
 - Support queue 1.2.0 (#177)
+- ConnectionListener interface for knowing the role changes of connections in
```

Comment:
https://en.wikipedia.org/wiki/Observer_pattern

But this is not really a `Listener`, because in our case there is a feedback. I've renamed it into `ConnectionObserver`, but I will be glad of any suggestions.

https://go.dev/doc/effective_go#interface-names

Reply:
I'll think about it again. I still don't really like `Observer`, but it's not better yet.

Reply:
Mayby `ConnectionRegister`?

Reply:
Maybe `Handler`...

Reply:
I've renamed it to `ConnectionHandler`. It is a shorter name at least.